src/charts/chartOptions.html

```diff
@@ -12,52 +12,52 @@
                         </div>
                     </div>
                     <div class="row">
                         <div class="cell">
                             <span class="vertical-bottom time">Minutes</span>
                         </div>
                         <div class="cell">
-                            <span class="vertical-bottom button" rv-each-timeperiod="timeperiod_arr | filter 'minutes'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
+                            <span class="vertical-bottom button fill-bg-color" rv-each-timeperiod="timeperiod_arr | filter 'minutes'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
                         </div>
                     </div>
                     <div class="row">
                         <div class="cell">
                             <span class="vertical-bottom time">Hours</span>
                         </div>
                         <div class="cell">
-                            <span class="vertical-bottom button" rv-each-timeperiod="timeperiod_arr | filter 'hours'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
+                            <span class="vertical-bottom button fill-bg-color" rv-each-timeperiod="timeperiod_arr | filter 'hours'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
                         </div>
                     </div>
                     <div class="row">
                         <div class="cell">
                             <span class="vertical-bottom time">Days</span>
                         </div>
                         <div class="cell">
-                            <span class="vertical-bottom button" rv-each-timeperiod="timeperiod_arr | filter 'days'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
+                            <span class="vertical-bottom button fill-bg-color" rv-each-timeperiod="timeperiod_arr | filter 'days'" rv-on-click="changeTimePeriod" rv-data-timeperiod="timeperiod.value">{timeperiod.digit}</span>
                         </div>
                     </div>
                 </div>
             </div>
-            
+
             <div class="cell">
                 <div class="chartOptions_button chart_type" rv-on-click="toggleChartTypeSelector"rv-class-button_selected="showChartTypeSelector" data-balloon="Chart type" data-balloon-pos="down">
-                    <div class="img"><img rv-src="chartType.value | append '.svg' | prepend 'images/'"></div>
+                    <div class="img"><img src='data:image/png;base64,R0lGODlhFAAUAIAAAP///wAAACH5BAEAAAAALAAAAAAUABQAAAIRhI+py+0Po5y02ouz3rz7rxUAOw==' rv-class="chartType.value | append '-icon'"></div>
```

Comment:
@negar-binary Aproov did similar changes and I recommend to use use DIV(rather than IMG). There are some other issues reported because of this img element. Please watch https://drive.google.com/open?id=0B-SvgdYmyOyKWU1EeEx4S09LbWs